web-sdk/v3.md

```diff
@@ -246,7 +260,266 @@ document.getElementById("open-form").addEventListener("click", function() {
+
+<table>
+<thead>
+  <tr>
+    <th>Parameter</th>
+    <th>Value</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td><code>logo.url</code></td>
+    <td>string</td>
+    <td>The URL of the logo on the iframe.</td>
+  </tr>
+  <tr>
+    <td><code>logo.style</code></td>
+    <td>CSS properties</td>
+    <td>The style of the logo on the iframe.</td>
+  </tr>
+  <tr>
+    <td><code>form.style</code></td>
+    <td>CSS properties</td>
+    <td>The style of the form element of the iframe.</td>
+  </tr>
+  <tr>
+    <td><code>overlay.style</code></td>
+    <td>CSS properties</td>
+    <td>The style of the overlay element of the iframe.</td>
+  </tr>
+  <tr>
+    <td><code>close.location</code></td>
+    <td>
+      string<br />
+      <em>Possible values: form | overlay | none</em><br />
+      <em>Default value: overlay</em>
+    </td>
+    <td>The location of the close icon.</td>
+  </tr>
+  <tr>
+    <td><code>close.color</code></td>
+    <td>string</td>
+    <td>The color of the close icon.</td>
+  </tr>
+  <tr>
+    <td><code>title.text</code></td>
+    <td>
+      string<br />
+      <em>Default value: "Connect your card"</em>
+    </td>
+    <td>The text of the title at the top of the page.</td>
+  </tr>
+  <tr>
+    <td><code>title.style</code></td>
+    <td>CSS properties</td>
+    <td>The style of the title at the top of the page.</td>
+  </tr>
+  <tr>
+    <td><code>cardNumber.label</code></td>
+    <td>LabelStyles (see definition below)</td>
+    <td>The style of the Card Number label.</td>
+  </tr>
+  <tr>
+    <td><code>cardNumber.input</code></td>
+    <td>InputStyle (see definition below)</td>
+    <td>The style of the Card Number input field.</td>
+  </tr>
+  <tr>
+    <td><code>expiryDate.label</code></td>
+    <td>LabelStyles (see definition below)</td>
+    <td>The style of the Expiry Date label.</td>
+  </tr>
+  <tr>
+    <td><code>expiryDate.input</code></td>
+    <td>InputStyle (see definition below)</td>
+    <td>The style of the Expiry Date input field.</td>
+  </tr>
+  <tr>
+    <td><code>country.label</code></td>
+    <td>LabelStyles (see definition below)</td>
+    <td>The style of the Country field label.</td>
+  </tr>
+  <tr>
+    <td><code>country.input</code></td>
+    <td>InputStyle (see definition below)</td>
+    <td>The style of the Country input field.</td>
+  </tr>
+  <tr>
+    <td><code>terms.text</code></td>
+    <td>
+      string<br />
+      <em>Default value: "I authorize &#123;&#123;scheme&#125;&#125; to monitor my payment card to identify transactions that qualify for a reward and for &#123;&#123;scheme&#125;&#125; to share such information with &#123;&#123;companyName&#125;&#125;, to enable my card linked offers and target offers that may be of interest to me."</em>
+    </td>
+    <td>The Terms of Use text.</td>
```

Comment:
I remember this was a tricky one, but the way it its there's nothing simpler and direct :ok_hand: